src/main/java/UseCase/RegisterArtistInteractor.java

```diff
@@ -0,0 +1,41 @@
+package UseCase;
+
+import Database.userAccessInterface;
+import Database.userDsData;
+import Entities.User;
+import Entities.userFactory;
+import InputBoundary.RegisterArtistInputBoundary;
+import InputData.RegisterArtistInputData;
+import OutputBoundary.loginOutputBoundary;
+
+public class RegisterArtistInteractor implements RegisterArtistInputBoundary{
+    private final loginOutputBoundary registerPresenter;
+    private final userFactory userFactory;
+
+    userAccessInterface allUsers;
```

Comment:
In case you weren't sure how to access the database (considering it isn't initialized --)
- Change to `private final userAccessInterface userList;` (or keep it as allUsers, doesn't really matter)
- In the initializer, `this.userList = Database.UserList.getInstance();`

Reply:
They are now updated, thanks!